docs/zetaclient/hotkey_tss_passwords.md

```diff
@@ -0,0 +1,34 @@
+## Hot Key and TSS key-share Passwords
+
+### Zetaclient
+Previously there were two environment variables being used to store passwords encrypting the tss key file and local operator keyring file.
+
+* HOTKEY_PASSWORD
+* TSS_FRAGMENT_SEED
+
+With this new change, these variables will no longer be valid.
```

Comment:
Need to precise the version where it changes.

I think we could mention this in the filename `12.2` -> `12.3`